apps/widget-configurator/src/app/configurator/consts.ts

```diff
@@ -31,7 +31,7 @@ export const DEFAULT_TOKEN_LISTS: TokenListItem[] = [
   { url: 'https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/set.tokenlist.json', enabled: false },
   { url: 'https://synths.snx.eth.link', enabled: false },
   { url: 'https://testnet.tokenlist.eth.link', enabled: false },
-  { url: 'https://gateway.ipfs.io/ipns/tokens.uniswap.org', enabled: false },
+  { url: 'https://cloudflare-ipfs.com/ipns/tokens.uniswap.org', enabled: false },
```

Comment:
Is there no ens for this instead?
That is, assuming the other `*.eth.link` domains work, we could use that instead of cloudflare since Elena is still seeing the CORS errors

Reply:
I don't know about that.
The only source I have is https://tokenlists.org/token-list?url=https://gateway.ipfs.io/ipns/tokens.uniswap.org.

So, I see the only reliable solution is to proxy the request.